dotnet-desktop-guide/net/winforms/controls-design/designer-differences-framework.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Designers changes from .NET Framework
+description: Learn about the Windows Forms designer changes from .NET Framework to .NET.
+ms.date: 05/01/2023
+ms.topic: overview
+no-loc: ["UserControl", "UserControl1", "UserControlProject", "Label", "Button", "Form", "TextBox"]
+dev_langs:
+- "csharp"
+- "vb"
+---
+
+# The designer changes since .NET Framework (Windows Forms .NET)
+
+The Visual Designer for Windows Forms for .NET has had some improvements and changes since .NET Framework. These changes largely affect custom control designers. This article describes the key differences from .NET Framework.
+
+[!INCLUDE [desktop guide under construction](../../includes/desktop-guide-preview-note.md)]
+
+Visual Studio is a .NET Framework-based application, and as such, the Visual Designer you see for Windows Forms is also based on .NET Framework. With a .NET Framework project, both the Visual Studio environment and the Windows Forms app being designed to run within the same process, **devenv.exe**. This poses a problem when you're working with a Windows Forms .NET (not .NET Framework) app. Both .NET and .NET Framework code can't work within the same process. As a result, Windows Forms .NET uses a different designer, the "out-of-process" designer.
+
+## Out-of-process designer
+
+The out-of-process designer is a process called **DesignToolsServer.exe**, and is run along-side Visual Studio's **devenv.exe** process. The **DesignToolsServer.exe** process runs on the same version and platform of .NET that your app has been setup to target, such as .NET 7 and x64.
```

Comment:
```suggestion
The out-of-process designer is a process called **DesignToolsServer.exe**, and is run along-side Visual Studio's **devenv.exe** process. The **DesignToolsServer.exe** process runs on the same version and platform of .NET that your app has been set 
```suggestion
The out-of-process designer is a process called **DesignToolsServer.exe**, and is run along-side Visual Studio's **devenv.exe** process. The **DesignToolsServer.exe** process runs on the same version and platform of .NET that your app has been setup to target, such as .NET 7 and x64.
```up to target, such as .NET 7 and x64.
```